lab-ron/test/test2-greetings.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
This is a good start but it needs to test something with an assertion, right now this doesn't test anything.